_pages/en/strategy-build-vs-buy.md

```diff
@@ -37,66 +44,140 @@ th {
-
-### Buy vs Build: oversimplifying solutions architecture
-
-Looking for a systemic and overarching approach to buy vs build of solutions at the organizational level means bypassing the whole concept of solution architecture, where the required experts of the organization, both from business lines, finance, security, information technology, etc., come together to figure out the best possible way of accomplishing its mission through leveraging its various resources including people, tools, processes and rules.
+- [Introduction](#introduction)
+  - [Purpose](#purpose)
+  - [Target Audience](#target-audience)
+  - [Business Case](#business-case)
+- [Guiding Policy](#guiding-policy)
+  - [Governance, Compliance, and Reporting](#governance-compliance-and-reporting)
+  - [Finance](#finance)
+  - [Architecture](#architecture)
+- [Coherent set of actions](#coherent-set-of-actions)
+- [Measuring the Strategy's success](#measuring-the-strategys-success)
+- [Appendix A - Business Case (Diagnostic)](#appendix-a---business-case-diagnostic)
+- [Appendix B - Traceability Matrix](#appendix-b---traceability-matrix)
+- [Appendix D - Definitions](#appendix-d---definitions)
+- [Appendix C - Context](#appendix-c---context)
+- [Appendix E - References](#appendix-e---references)
+
+## Introduction
+
+### Purpose
+
+To provide IITB with a build versus buy assessment framework.
+
+The strategy includes:
+
+1. A **guiding policy**, which serves to set automatic decisions that defines the build versus buy assessment framework
+2. A **coherent set of actions** (an action plan), which serve to operationalize the build versus buy assessment framework.
+
+The intent behind this strategy is to communicate a decision by the CIO (not yet approved) on a path forward (the Guiding Policy), and what investments are needed to operationalize that decision (the coherent set of actions).
+
+### Target Audience
+
+This **strategy document** is targeted to stakeholders involved in determining whether to build or to buy software components.
+More specifically, stakeholders involved in architecting solutions, defining technology standards, developing, delivering, operationalizing and maintaining IT Solutions (such as Enterprise Architecture, Enterprise Operations, etc.).
+This includes both IITB and non-IITB stakeholders (such as from Business Functions, SSPB, CDO, ISB, IAERMB, and SSC).
+The list of stakeholders are listed in section [Coherent set of actions](#coherent-set-of-actions) and are expected to participate in the execution stage of this strategy necessary to operationalize the Guiding Policy.
+
+The **Guiding Policy**, once operationalized, will target stakeholders involved when the need for an IT Solution is raised, when the IT Solution is conceptualized, architected, developed, delivered, operationalized, and maintained.
+All ESDC personnel involved in the exploration of IT Solutions and in IT investment decisions are expected to adhere to this policy.
+
+### Business Case
```

Comment:
Made a few tweaks to business case, mainly an intro to lead to the problem statement. Will have a second pass to make clearer.

Will also need to look at the problem statements.

Reply:
Just a suggestion
In Business case: 
The first sentence: (little tweak)
The COVID-19 crisis forced the acceleration of moving to the digital age which requires improving IT’s responsiveness......

Reply:
The business case section is looking good!  I wonder if we need the first sentence of the second paragraph. I think it reads fine without it and it might be more clear as it mentions responsiveness but then responsiveness is not one of the listed benefits.

Reply:
I like the idea of adding the COVID as a catalyst. I think I'll move it to the context document?